tests/ft_integration/Tribe/Tickets/__snapshots__/MetaboxTest__test_get_panels__post with RSVP__0.snapshot.html

```diff
@@ -6,19 +6,7 @@
 >
 	<div class="tribe_sectionheader ticket_list_container">
 		<div class="ticket_table_intro">
-			<button
```

Comment:
This looks like a side-effect of Classy being enabled. Disabling Classy for the suite, these should not change.